lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -602,6 +615,17 @@ defmodule Phoenix.LiveViewTest do
     end
   end
 
+  defmacro assert_redirect(view, to, flash, func) do
```

Comment:
I am thinking we should change this API, as it is getting complex with the number of arguments, and no longer receive a function. Make it instead:

```
defmacro assert_redirect(view, to, flash)
```

The same way `assert_remove` works.